docs/sources/tutorials/alerting-get-started/index.md

```diff
@@ -32,7 +32,19 @@ In this tutorial you will:
 - Set up an alert rule.
 - Receive firing and resolved alert notifications in a public webhook.
 
-Check out [Part 2](http://grafana.com/tutorials/alerting-get-started-pt2/) if you want to learn more about alerts and notification routing.
+<!-- INTERACTIVE ignore START -->
+
+{{ <admonition type="tip">}}
+Before you dive in, remember that you can [explore advanced topics like alerts and notification routing](http://grafana.com/tutorials/alerting-get-started-pt2/) in the second part of this guide.
+{{ </admonition >}}
+
+<!-- INTERACTIVE ignore END -->
+
+{{< docs/ignore >}}
+
+Before you dive in, remember that you can [explore advanced topics like alerts and notification routing](http://grafana.com/tutorials/alerting-get-started-pt2/) in the second part of this guide.
```

Comment:
alert routing? or alerts in general? maybe rules or instances? in either case, being more specific would help:)

Reply:
i keep forgetting , good point! it should be alert instances

Reply:
done, thanks @brendamuir !